.github/workflows/wc-plan.yaml

```diff
@@ -18,7 +18,7 @@ on:
 jobs:
   plan:
     name: "plan (${{matrix.target.target}})"
-    runs-on: ${{fromJSON(matrix.target.runs-on)}}
+    runs-on: ${{matrix.target.runs_on}}
```

Comment:
We believe two issues were at play here.
- `runs-on`  doesn't seem to exist here
- Strings like "ubuntu-latest" don't seem to parse in `fromJSON` maybe??